tests/clients/boto3/test_boto3_lambda.py

```diff
@@ -1,95 +1,96 @@
         self.function_name = "myfunc"
-
-    def tearDown(self):
+        yield
         # Stop Moto after each test
         self.mock.stop()
         agent.options.allow_exit_as_root = False
 
-    def test_lambda_invoke(self):
-        with tracer.start_active_span('test'):
+    def test_lambda_invoke(self) -> None:
+        with tracer.start_as_current_span("test"):
             result = self.aws_lambda.invoke(FunctionName=self.function_name, Payload=json.dumps({"message": "success"}))
 
-        self.assertEqual(result["StatusCode"], 200)
+        assert result["StatusCode"] == 200
         result_payload = json.loads(result["Payload"].read().decode("utf-8"))
-        self.assertIn("message", result_payload)
-        self.assertEqual("success", result_payload["message"])
+        assert "message" in result_payload
+        assert "success" == result_payload["message"]
 
         spans = self.recorder.queued_spans()
-        self.assertEqual(2, len(spans))
+        assert 2 == len(spans)
 
         filter = lambda span: span.n == "sdk"
         test_span = get_first_span_by_filter(spans, filter)
-        self.assertTrue(test_span)
+        assert test_span
 
         filter = lambda span: span.n == "boto3"
         boto_span = get_first_span_by_filter(spans, filter)
-        self.assertTrue(boto_span)
+        assert boto_span
 
-        self.assertEqual(boto_span.t, test_span.t)
-        self.assertEqual(boto_span.p, test_span.s)
+        assert boto_span.t == test_span.t
+        assert boto_span.p == test_span.s
 
-        self.assertIsNone(test_span.ec)
-        self.assertIsNone(boto_span.ec)
+        assert test_span.ec is None
+        assert boto_span.ec is None
 
-        self.assertEqual(boto_span.data['boto3']['op'], 'Invoke')
+        assert boto_span.data['boto3']['op'] == 'Invoke'
         endpoint = f'https://lambda.{self.lambda_region}.amazonaws.com'
-        self.assertEqual(boto_span.data['boto3']['ep'], endpoint)
-        self.assertEqual(boto_span.data['boto3']['reg'], self.lambda_region)
-        self.assertIn('FunctionName', boto_span.data['boto3']['payload'])
-        self.assertEqual(boto_span.data['boto3']['payload']['FunctionName'], self.function_name)
-        self.assertEqual(boto_span.data['http']['status'], 200)
-        self.assertEqual(boto_span.data['http']['method'], 'POST')
-        self.assertEqual(boto_span.data['http']['url'], f'{endpoint}:443/Invoke')
-
-    def test_lambda_invoke_as_root_exit_span(self):
+        assert boto_span.data['boto3']['ep'] == endpoint
+        assert boto_span.data['boto3']['reg'] == self.lambda_region
+        assert 'FunctionName' in boto_span.data['boto3']['payload']
+        assert boto_span.data['boto3']['payload']['FunctionName'] == self.function_name
+        assert boto_span.data['http']['status'] == 200
+        assert boto_span.data['http']['method'] == 'POST'
+        assert boto_span.data['http']['url'] == f'{endpoint}:443/Invoke'
```

Comment:
This whole block is using single quotes on strings. Please, update to double quotes in the rest of the file and the other test files.

```
assert boto_span.data["boto3"]["op"] == "Invoke"
```

Reply:
Done.